lib/engaging_networks_rest/client.rb

```diff
@@ -9,7 +9,7 @@ module EngagingNetworksRest
   class Client
     attr_reader :api_key, :connection, :ens_auth_key
 
-    ENS_DOMAIN = 'www.e-activist.com'
+    ENS_DOMAIN = 'ca.engagingnetworks.app'
```

Comment:
Should we avoid setting a default that may not work to force gem's users to set a value explicitly?

Reply:
Yeah, that's valid. We already require setting an API key, so it seems reasonable to also require setting a hostname. (And it's a breaking change either way!) Will update.